### PR TITLE
fix: docs/framework/unmanaged-api/wmi/putinstancewmi.md

### PR DESCRIPTION
**Pull request by nschonni**
- writen -> written
- storedwith -> stored with

### CODE REVIEW

File: docs/framework/unmanaged-api/wmi/putinstancewmi.md [hunk 1]
```diff
@@ -2,57 +2,59 @@
...
-| `WBEM_FLAG_USE_AMENDED_QUALIFIERS` | 0x20000 | If set, WMI does not store any qualifiers with the **Amended** flavor. <br> If not set, it is assumed that this object is not localized, and all qualifiers are storedwith this instance. |
+| `WBEM_FLAG_USE_AMENDED_QUALIFIERS` | 0x20000 | If set, WMI does not store any qualifiers with the **Amended** flavor. <br> If not set, it is assumed that this object is not localized, and all qualifiers are stored with this instance. |
 | `WBEM_FLAG_CREATE_OR_UPDATE` | 0 | Create the instance if it does not exist, or overwrite it if it exists already. |
 | `WBEM_FLAG_UPDATE_ONLY` | 1 | Update the instance. The instance must exist for the call to be successful. |
 | `WBEM_FLAG_CREATE_ONLY` | 2 | Create the instance. The call fails if the instance already exists. |
 | `WBEM_FLAG_RETURN_IMMEDIATELY` | 0x10 | The flag causes a semisynchronous call. |
 
-`pCtx`  
-[in] Typically, this value is `null`. Otherwise, it is a pointer to an [IWbemContext](/windows/desktop/api/wbemcli/nn-wbemcli-iwbemcontext) instance that can be used by the provider that is providing the requested classes. 
+`pCtx`
```
Comment (mairaw) [inline docs/framework/unmanaged-api/wmi/putinstancewmi.md]:
```suggestion
`pCtx`\
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Changes look good @nschonni. Again just the parameters lost their soft return.

File: docs/framework/unmanaged-api/wmi/putinstancewmi.md (hunk 1 above)
Comment (nschonni) [reply docs/framework/unmanaged-api/wmi/putinstancewmi.md]: I missed that you were using the double space return. I'm wondering if these should these be H3s in these docs?

Comment (mairaw) [reply docs/framework/unmanaged-api/wmi/putinstancewmi.md]: It's something worth investigating. I was going for a similar look and feel of the managed reference pages.

Comment (mairaw) [reply docs/framework/unmanaged-api/wmi/putinstancewmi.md]: Also, the double space return is a migration hack that we're trying to get rid of over time. So I like that we're cleaning the extra spaces.

**Review by mairaw — APPROVED** [review]
I'll fix my suggestion in PR 11012 instead. We can discuss a different formatting for those as well. Thanks!

**Pull request merged**